sql/driver/driver_test.go

```diff
@@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
 	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
 		t.Fatal(err)
 	}
+	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
```

Comment:
If it isn't persistent it will return an error (see the error above that
gets returned when a table is created against no database). You are right
that we don't check whether it didn't create the table against another
database (corruption or misbehavior scenario).

IMHO a lot of the testing that were doing at the moment is extremely
barebones. I suppose it's because we're still playing around a bit.

On Fri, Jul 10, 2015 at 2:45 PM Tamir Duberstein <notifications@github.com>
wrote:

> In sql/driver/driver_test.go
> <https://github.com/cockroachdb/cockroach/pull/1660#discussion_r34384012>:
>
> > @@ -149,6 +149,9 @@ func TestCreateTable(t *testing.T) {
> >  	if _, err := db.Exec("CREATE TABLE IF NOT EXISTS users " + cols); err != nil {
> >  		t.Fatal(err)
> >  	}
> > +	if _, err := db.Exec("CREATE TABLE employees " + cols); err != nil {
>
> Doesn't this still need a test that confirms that the session was indeed
> persisted on the connection between statements? in other words, I think we
> need to confirm that these databases were created on t.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1660/files#r34384012>.
>